src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -308,6 +307,66 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
     }
   }
 
+  /** Fetches and stores all delegates from the latest blocks stored in the database. */
```

Comment:
I'm not sure why it's here in that PR. This code is already in master.

Reply:
Where? I don't see it...
https://github.com/Cryptonomic/Conseil/blob/master/src/main/scala/tech/cryptonomic/conseil/Lorre.scala#L292